test/controller/objecting.entity.name.controller.spec.unit.ts

```diff
@@ -264,6 +293,31 @@ describe("objecting entity name tests", () => {
     expect(response.status).toEqual(500);
     expect(response.text).toContain(ERROR_500);
   });
+
+  it("should navigate to check you answers and send details to mongo when check answers", async () => {
```

Comment:
Perhaps: "should navigate to check you answers and send details to mongo when check answers flag set to true"